api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
@nikhiljindal Swagger 1.2 spec does not support that (multiple args of the same name) based on my knowledge.

[Swagger 2.0](https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#fixed-fields-7) spec supports that with `collectionFormat` using `multi`:

```
multi - corresponds to multiple parameter instances instead of multiple values 
for a single instance foo=bar&foo=baz. This is valid only for parameters in 
"query" or "formData"
```

cc @webron - the expert in Swagger Spec